lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -137,12 +145,12 @@
 
     dns_registration:
         description:
-            - If true web app hostname is not registered with DNS on creation.
+            - Whether or not the web app hostname is registered with DNS on creation. Set to C(false) to register.
         type: bool
 
     skip_custom_domain_verification:
         description:
-            - If true, custom (non *.azurewebsites.net) domains associated with web app are not verified.
+            - Whether or not custom (non *.azurewebsites.net) domains associated with web app are not verified. Set to C(true) to skip.
```

Comment:
Ugh. This one still sounds weird - it's hard to tell what the "not" applies to. Let's make the description match the parameter name:
```suggestion
            - Whether or not to skip verification of custom (non *.azurewebsites.net) domains associated with web app. Set to C(true) to skip.
```